documentation/modules/proc-applying-infra-config.adoc

```diff
@@ -20,15 +17,15 @@ ifeval::["{cmdcli}" == "oc"]
 {cmdcli} login -u developer
 ----
 
-. Select the project where {ProductName} is installed:
+. Change the project to the project where {ProductName} is installed:
```

Comment:
Reads clumsily to me.

Reply:
@k-wall, agreed. Trying to come up with a better phrasing, but this one is tough. Does "Chang to the project where {ProductName} is installed" read better to you?

Reply:
Yes, it does.